HotFix/Transport/TcpTransport.cs

```diff
@@ -33,17 +34,32 @@ public static TcpTransport Initiate(IClock clock, string address, int port)
         public static TcpTransport Accept(IClock clock, string address, int port)
         {
             var endpoint = new IPEndPoint(IPAddress.Parse(address), port);
-            var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp) { ReceiveTimeout = 1000, NoDelay = true };
-
-            socket.EnableFastPath();
-            socket.Bind(endpoint);
-            socket.Listen(1);
-
-            var client = socket.Accept();
-
-            socket.Dispose();
-
-            return new TcpTransport(clock, client);
+            using (var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp) { ReceiveTimeout = 1000, NoDelay = true })
```

Comment:
Not sure that the `using` adds any value if we go with the `EndAccept` approach?